lib/category_utils.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CategoryUtils
+  # Removing prefixes for pages outside mainspace
+  # Everything till (and including) the first semicolon(:) is removed
+  def self.get_titles_without_prefixes(pages)
```

Comment:
It would be useful to document in a comment what a `page` object looks like here and why. (ie, I guess it's a hash that includes at least `title` and `ns` keys, which you can get from a `list=categorymembers` query from the MediaWiki API)

Reply:
Yes, documented it in a comment